web/src/utils/getNavigationDeepLinks.ts

```diff
@@ -0,0 +1,21 @@
+import { LocationModel } from 'api-client'
+
+export const getNavigationDeepLinks = (location: LocationModel, fallback: string): string => {
+  if (!location.location || !location.coordinates) {
+    return fallback
+  }
```

Comment:
I think we should always use `geo` or `maps` uri scheme, even if there are no coordinates/location. If there is no location, it should be easy (just don't pass a query). If there are no coordinates, we can just pass `0,0` to search around the current location (https://en.wikipedia.org/wiki/Geo_URI_scheme#Unofficial_extensions). Not sure about iOS though, the `geo` scheme is not supported there, is it?

Reply:
`geo` is a newer scheme that is not yet supported by ios, so we have ti use different schemes.

The fallback is also used for desktop devices and we could use desktop maps apps for macOS/Linux/Windows (schema) but thats not very common.
I disagree with your solution referring to the link you posted, because the coordinates may lead to the correct address but not to the correct POI and people will be confused if another description will be presented.
I guess most people creating pois using sth like google maps to get address and names, therefore `location.location` should work.
You can test it by `return `geo:0,0?q=${lat},${long}`  for `isAndroid` and open `bfz gGmbh`
GoogleMaps: Show correct address with wrong description (several places at this address)
MapsMe: works with both schema solutions

Reply:
pois without coordinates and address gonna be filter in IGAPP-985, so i keep the current implementation